File: scripts/switch_main.py
import subprocess
import os

os.chdir('/vercel/share/v0-project')

try:
    print('Fetching latest changes...')
    subprocess.run(['git', 'fetch', 'origin'], check=True)
    
    print('Switching to main branch...')
    subprocess.run(['git', 'checkout', 'main'], check=True)
    
    print('Successfully switched to main branch!')
except subprocess.CalledProcessError as e:
    print(f'Error switching branch: {e}')
    exit(1)

File: scripts/switch-main.mjs
import { execSync } from 'child_process';
import { chdir } from 'process';

chdir('/vercel/share/v0-project');

try {
  console.log('Fetching latest changes...');
  execSync('git fetch origin', { stdio: 'inherit' });
  
  console.log('Switching to main branch...');
  execSync('git checkout main', { stdio: 'inherit' });
  
  console.log('Successfully switched to main branch!');
} catch (error) {
  console.error('Error switching branch:', error.message);
  process.exit(1);
}
